ArchiSteamFarm.Tests/SteamChatMessage.cs

```diff
@@ -176,6 +176,9 @@ public sealed class SteamChatMessage {
 			Assert.AreEqual(ContinuationCharacter + longLine, output[3]);
 		}
 
+		[TestMethod]
+		public void ReservedSizeForEscapingIsProperlyCalculated() => Assert.AreEqual(ReservedEscapeMessageBytes, Encoding.UTF8.GetByteCount(@"\") + 4);
```

Comment:
Please add a comment explaining why this is `4` and not `Encoding.UTF8.GetMaxByteCount(1)`.